### PR TITLE
[hotfix] remove unused methods

### PR DESCRIPTION
**Pull request by xuzifu666**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by xuzifu666** [comment]
a little hot fix, have a review plz  cc @wanpengfei-git

File: fe/fe-core/src/main/java/com/starrocks/analysis/DescriptorTable.java [hunk 1]
```diff
@@ -44,7 +44,6 @@
  * them unique ids..
  */
 public class DescriptorTable {
-    private static final Logger LOG = LogManager.getLogger(DescriptorTable.class);
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/analysis/DescriptorTable.java:47]: import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

should be removed too

Comment (xuzifu666) [reply fe/fe-core/src/main/java/com/starrocks/analysis/DescriptorTable.java:47]: get，thanks

**Comment by xuzifu666** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Pull request merged**